src/app/layout.tsx

```diff
@@ -0,0 +1,122 @@
+"use client";
+import React from "react";
+import "./global.css";
+import { useState } from "react";
+import { Dialog, DialogPanel } from "@headlessui/react";
+import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/outline";
+import LogoIcon from "@/components/LogoIcon";
+import { LocalizationProvider } from "@mui/x-date-pickers";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { Provider } from "react-redux";
+import { store } from "@/redux/store";
+import Link from "next/link";
+
+const navigation = [
+  { name: "Fills", href: "fills" },
+  { name: "Visual", href: "visual" },
+  { name: "History", href: "history" },
+  { name: "About", href: "about" },
+];
+
+export default function Layout({ children }: { children: React.ReactNode }) {
+  const [mobileMenuOpen, setMobileMenuOpen] = useState(false);
+
+  return (
+    <html lang="en">
+      <body>
+        <header className="bg-white">
+          <nav
+            aria-label="Global"
+            className="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8"
+          >
+            <div className="flex lg:flex-1">
+              <Link href="/" className="-m-1.5 p-1.5">
+                <span className="sr-only">Your Company</span>
+                <LogoIcon />
```

Comment:
I havent gotten there yet but Im assuming you fixed the icon portion that I loaded incorrectly before